### PR TITLE
Feature/genres page

### PR DESCRIPTION
**Pull request by mrunalshri**

### CODE REVIEW

File: src/Components/cardRow/CardRow.tsx [hunk 1]
```diff
@@ -34,9 +34,12 @@ export const CardRow: React.FC<CardRowInterface> = ({ movies, title }) => {
       <>
         {movies.map((movie: Movie) => (
           <React.Fragment key={`movie-data-${movie.id}`}>
-            {(movie.genre_ids.includes(parseInt(filter)) || filter === "A") && (
-              <MovieData movieDetails={movie} title={title} />
-            )}
+            {/* added extra check for inappropriate movie poster id: 1087040 */}
+            {movie.id !== 1087040 &&
```
Comment (mrunalshri) [inline src/Components/cardRow/CardRow.tsx:38]: @dingbat91 Please review this line for extra check

File: src/Components/Header/header.test.tsx [hunk 2]
```diff
@@ -7,13 +7,13 @@ jest.mock("react-router-dom", () => ({
 }));
 test("renders Header component correctly", () => {
   render(<Header />);
-  const headerComponent = screen.getByAltText("Man Eating Popcorn");
```
Comment (dingbat91) [inline src/Components/Header/header.test.tsx:10]: Was this an intentional change?

File: src/Components/Header/header.test.tsx [hunk 3]
```diff
@@ -7,13 +7,13 @@ jest.mock("react-router-dom", () => ({
...
+  const headerComponent = screen.getByLabelText("logo");
```
Comment (dingbat91) [inline src/Components/Header/header.test.tsx:10]: Was this an intentional change?

Comment (dingbat91) [reply src/Components/Header/header.test.tsx:10]: No totally fine. Just wanted to check it wasn't an unexpected change from a merge artifact from something. I see no issue in changing it. :)

File: src/Components/cardRow/CardRow.tsx (hunk 1 above)
Comment (dingbat91) [reply src/Components/cardRow/CardRow.tsx:38]: All good!

**Review by dingbat91 — APPROVED** [review]
Tested and runs without any disruption! Approved!

**Pull request merged**